src/tools/dotnet.lua

```diff
@@ -286,7 +297,7 @@
 		table.insert(flags, '/noconfig')
 
 		if cfg.project.icon then
-			local fn = project.getrelative(cfg.project, cfg.project.icon)
+			local fn = dotnet.getrelative(cfg.project, cfg.project.icon)
```

Comment:
Results in `project.getrelative(cfg.project.project, cfg.project.icon)`, so it should probably be
```suggestion
			local fn = dotnet.getrelative(cfg, cfg.project.icon)
```
Apply to all other places too.

if not, then 

```
function dotnet.getrelative(cfg, value)
```
should probably be

```
function dotnet.getrelative(project, value)
```

Reply:
> Thanks for the review, surprised the test suite did not catch anything. 😱

Possibly that `cfg.project.project` is also `cfg.project`; Testsuite uses indirectly `gcc.getrelative`, so it behave correctly.
BTW, `gcc.getrelative` has same "issue".

> Also this would have probably have easily been caught with your test suite.

Both can catch this.

> Has there been any discussion in getting that integrated recently?

Premake's team doesn't want integration tests, as they think it is too much work and maintenance (I disagree with them).